lib/Image.php

```diff
@@ -133,16 +133,8 @@ protected function get_dimensions( $dim ) {
 			'Image::get_dimension',
 			'2.0.0'
 		);
-		if ( isset($this->_dimensions) ) {
-			return $this->get_dimensions_loaded($dim);
-		}
-		if ( file_exists($this->file_loc) && filesize($this->file_loc) ) {
-			list($width, $height) = getimagesize($this->file_loc);
-			$this->_dimensions = array();
-			$this->_dimensions[0] = $width;
-			$this->_dimensions[1] = $height;
-			return $this->get_dimensions_loaded($dim);
-		}
+		$ds = array($this->width(), $this->height());
```

Comment:
Call me picky, but can we cut down the use of abbreviations in variable names? I will only know that `$ds` is supposed to mean if look at what kind of array it is going to build. I’d rather have `$dimensions` as a variable name.

Reply:
This should be
`return array($this->width(), $this->height());`

Reply:
> This should be
> `return array($this->width(), $this->height());`

Even better, yes!

Reply:
done!